client/src/components/ratingsReviews/ratingBreakdown.css

```diff
@@ -0,0 +1,4 @@
+.averageRating {
```

Comment:
You should probably be more specific when naming your classes. Even though you have your component CSS in a separate file, it will not remain isolated. If `averageRating` were to overlap with another component the definition last processed takes precedence, which could be all but impossible to debug.